### PR TITLE
Apache nifi faq

### PR DESCRIPTION
**Pull request by jpapejr**
Adds a FAQ entry for running Apache Nifi in an Istio Mesh

### CODE REVIEW

File: content/en/faq/applications/nifi.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: Can I run Apache Nifi inside an Istio mesh?
```
Comment (rvennam) [inline content/en/faq/applications/nifi.md:2]:
```suggestion
title: Can I run Apache NiFi inside an Istio mesh?
```

File: content/en/faq/applications/nifi.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+description: How to run Apache Nifi with Istio.
```
Comment (rvennam) [inline content/en/faq/applications/nifi.md:3]:
```suggestion
description: How to run Apache NiFi with Istio.
```

Comment (jpapejr) [reply content/en/faq/applications/nifi.md]: done, thx

File: content/en/faq/applications/nifi.md (hunk 1 above)
Comment (jpapejr) [reply content/en/faq/applications/nifi.md]: Corrected

**Comment by jpapejr** [comment]
Lint failures look to all be spelling issues which I thought would be addressed by adding `NiFi` to the .spelling file in https://github.com/istio/istio.io/pull/7266/commits/6f5f08c0eec1b7490ea2cef451e515976da6b123

Did I miss something?

**Pull request merged**